app/views/volunteers/_form.html.slim

```diff
@@ -0,0 +1,115 @@
+= simple_form_for(@volunteer) do |f|
+  - if f.error_notification.present?
+    .row
+      .col-xs-12
+        = f.error_notification
+
+  .row
+    .col-xs-12.col-md-6
+        = f.input :first_name
+
+    .col-xs-12.col-md-6
+        = f.input :last_name
+
+  .row
+    .col-xs-12.col-md-6
+      = f.input :date_of_birth, as: :date, start_year: Date.today.year - 90,
+        end_year: Date.today.year, order: [:day, :month, :year], include_blank: true
```

Comment:
I found how this is done. I have to define the order in the local files. For the en do we follow the GB or US format?